app/poros/obituary.rb

```diff
@@ -0,0 +1,14 @@
+class Obituary
+  attr_reader :headline,
+              :abstract,
+              :web_url,
+              :pub_date,
+              :word_count
+  def initialize(obituary_data)
+    @headline = obituary_data[:headline]
+    @abstract = obituary_data[:abstract]
+    @web_url = obituary_data[:web_url]
+    @pub_date = obituary_data[:pub_date]
+    @word_count = obituary_data[:word_count]
```

Comment:
I wasn't sure of the best way to go about this... I ended up consuming the NYtimes API and using this poro to pass all of the data from NYtimes to the controller/sinatra class which then passes that data as a response.

Reply:
Seems like a good strategy to me.

Reply:
I agree, it seems like you are following best practices from rails which is the best we know!